test/tools/unified-spec-runner/schema.ts

```diff
@@ -236,3 +236,8 @@ export interface ExpectedError {
   errorLabelsOmit?: string[];
   expectResult?: unknown;
 }
+
+/**
+ * A type that represents the test filter provided to the unifed runner.
+ */
+export type TestFilter = (test: Test) => null | string;
```

Comment:
I would say my personal preference when filtering on truthy/falsey values in this case would to actually return `false` instead of `null` for the filter. To me `false` is an explicit return value that says we expect this not to be filtered out, whereas `null` or even `undefined` are not explicit just non-existent. But I'm happy to lgtm this if anyone feels more strongly about it.